builder/scaleway/config.go

```diff
@@ -98,10 +125,43 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 			c.Organization = os.Getenv("SCALEWAY_API_ACCESS_KEY")
 		}
 	}
+	if c.Organization != "" {
+		warnings = append(warnings, "organization_id is deprecated in favor of project_id")
+		c.ProjectID = c.Organization
+	}
 
 	if c.Token == "" {
 		c.Token = os.Getenv("SCALEWAY_API_TOKEN")
 	}
+	if c.Token != "" {
+		warnings = append(warnings, "token is deprecated in favor of secret_key")
+		c.SecretKey = c.Token
+	}
+
+	if c.Region != "" {
+		warnings = append(warnings, "region is deprecated in favor of zone")
+		c.Zone = c.Region
+	}
+
+	if c.AccessKey == "" {
```

Comment:
would it make sense to use `scw.LoadEnvProfile()` for all env variables?

Reply:
Hum yep we could add the `Profile` to the config 🤔

Reply:
It may complicate a bit tis part. I think keeping in the way it is is the easier approach